src/js/schemas/service-schema/EnvironmentVariables.js

```diff
@@ -1,7 +1,12 @@
 import {Hooks} from 'PluginSDK';
+import {Link} from 'react-router';
 
 let EnvironmentVariables = {
-  description: 'Set environment variables for each task your service launches in addition to those set by Mesos.',
+  description: (
+    <span>
+    Set variables for each task your service launches. You can also use variables to expose Secrets. <Link to "system-security-secrets">Manage secrets here</Link>. <a href="https://mesosphere.github.io/marathon/docs/task-environment-vars.html">Learn more about variables</a>.
```

Comment:
This line should also be indented once further, and let's use `target="_blank" on the external link:
`<a href="https://mesosphere.github.io/marathon/docs/task-environment-vars.html" target="_blank">`